ca_tools/scripts/line_action_client.py

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env python
+
+import rospy
+import ca_tools.msg
+import actionlib
+
+
+def line_follower_client():
+
+    client = actionlib.SimpleActionClient(
+        'linefollower', ca_tools.msg.linefollowerAction)
+    
+    _duration_threshold = rospy.get_param("duration_threshold")
+    _times_oop = rospy.get_param("times_oop")
```

Comment:
Document this variable or name it with a more representative name

Reply:
Adressed